### DISCUSSION TITLE
Freya - Visual difference <p:card/> and <div class="card"/>

### DISCUSSION THREAD

**Discussion by brunoguadagnini** [primefaces/community#2061] [PrimeFaces Templates] [answered]
I would like to use the primefaces component `<p:card />` to have the same visual behavior as `<div class="card" />`, but apparently they are different. This can be checked on the demo page https://primefaces.org/freya/panel.xhtml

Is this a bug? Or is it something on purpose?

thanks.

![image](https://github.com/primefaces/community/assets/9530283/d2a6aa7e-2bc3-48b8-8e86-1f7d7d29ca32)

**Comment by melloware** [comment, marked as answer]
Simply update your Freya variables to match the Card border radius.

For example the default variable might be `$borderRadius: 4px !default;`

Simply change that to match what card has and regenerate your Freya theme.
